invenio/modules/tags/models.py

```diff
@@ -257,7 +257,8 @@ def __init__(self, bibrec=None, **kwargs):
         """TODO."""
         super(WtgTAGRecord, self).__init__(**kwargs)
 
-        self.bibrec = bibrec
+        if bibrec is not None:
```

Comment:
The error is here: https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/tags/testsuite/test_tags_restful.py#L491

If you follow the test execution with debug, you'll finish here ([creating a Tag linked with a Record](https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/tags/api.py#L207))

Here create the connection [Tag <--> Record](https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/tags/api.py#L237)
And it's when the `__init__` is executed and also `self.bibrec = None` that it'll generate the error... XD

btw, @jirikuncar started to [move the module outside](https://github.com/inveniosoftware/invenio-tags)..